subiquity/tests/api/test_api.py

```diff
@@ -590,7 +590,11 @@ async def test_guided_v2_resize_logical(self):
 class TestCore(TestAPI):
     @timeout()
     async def test_basic_core_boot(self):
-        cfg = 'examples/simple.json'
+        cfg = self.machineConfig('examples/simple.json')
+        with cfg.edit() as data:
+            attrs = data['storage']['blockdev']['/dev/sda']['attrs']
+            # The partition layout was bugged for disk sizes < 28GiB
+            attrs['size'] = str(25 << 30)
```

Comment:
I don't really understand this comment.

Reply:
Hopefully this commit message is clearer: https://github.com/canonical/subiquity/pull/1674/commits/19ef465d48dcaafcd385103885bfe9497c4dba1e